test/utils.js

```diff
@@ -18,6 +18,15 @@ test('merge()', function (t) {
     var nestedArrays = utils.merge({ foo: ['baz'] }, { foo: ['bar', 'xyzzy'] });
     t.deepEqual(nestedArrays, { foo: ['baz', 'bar', 'xyzzy'] });
 
+    var setCount = 0;
+    var observed = [];
+    Object.defineProperty(observed, 0, {
```

Comment:
This should be inside a subtest that's skipped when `Object.defineProperty` is unavailable (for ES3 environments)

Reply:
Let me know how that looks: c213b6bf15a22a9090a7a186401cbc0e75b0f23f

Reply:
Close! I'll comment on the commit.